### PR TITLE
docs: update example scripts in Introduction to scripted browser monitors

### PR DESCRIPTION
**Pull request by shevaun-nr**
docs: update example scripts in Introduction to scripted browser monitors

Scripts are formatted for legacy runtime, so this updates them to use $webDriver/$selenium and await/async. Some of the scripts are a bit more complex to convert; where this is the case they've been tested to ensure they run from a node command line - but please feel free to retest.

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx [hunk 1]
```diff
@@ -33,29 +33,36 @@ For a look at how to use the Selenium IDE to record your workflows and export th
...
-    return $browser.findElement($driver.By.linkText("Configuration Panel"));
+await $webDriver.get("https://my-website.com");
+await $webDriver.findElement($selenium.By.linkText("Configuration Panel"));
+```
+
+You can also wrap each action in a `then(function(){})` call, but in that case the wrapped function must `return` each asynchronous function to ensure they complete before the script moves on: 
+
+```
+$webDriver.get("https://my-website.com").then(function(){
+    return $webDriver.findElement($selenium.By.linkText("Configuration Panel"));
```
Comment (shevaun-nr) [inline src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:55]: Very not sure if this should be. left in or not. Technically still a valid approach as long as the Promises are returned?

File: src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx [hunk 2]
```diff
@@ -222,14 +229,13 @@ To avoid these issues, set wait conditions that will pause the script until the
...
-      return $browser.getTitle().then(function(title) {
-    //Specify the expected page title.
-        return title === "Your Page Title Here";
-        });
+    await $webDriver.wait(async function() {
+      //Tell the monitor to get the page title.
+      var title = await $webDriver.getTitle();
+      //Specify the expected page title.
+      return title === "React app";
     //If the condition is not satisfied within a timeout of 1000 milliseconds (1 second), proceed anyway.
```
Comment (ryanv94) [inline src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:237]: @shevaun-nr Should this comment also be updated to 3000ms / 3s ?

Comment (shevaun-nr) [reply src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx:237]: Oh, actually it's the other way around - the 3000 ms timeout is left over from when I was testing the script locally. Ugh, also the 'React App' title, I thought I had changed that. Thanks for spotting it!

**Comment by shevaun-nr** [comment]
Closing this PR as I've opened a 'real' PR against the main repo instead: https://github.com/newrelic/docs-website/pull/16871

**Pull request closed without merging**